docs/contributing.rst

```diff
@@ -0,0 +1,3 @@
+Contributing
+============
+
```

Comment:
We should add something here, or link to the tequila docs for contributing (I think that's probably better/more maintainable).